### ISSUE TITLE
Cannot Generate specific critical path css for pages - Server issue

### ISSUE THREAD

**Issue by jazir555** [wp-media/wp-rocket#4237]
**Describe the bug**
The same server bug I reported before about being unable to generate critical path css for specific pages is occurring again.

**To Reproduce**
1. Go to a page
2. Generate specific cpcss
3. The generation of cpcss will fail

**Expected behavior**
Critical path css generated correctly.

Why does this keep occurring?

**Comment by jazir555** [comment]
Still having issues with the generate specific cpcss feature. Sometimes it is saying the cpcss is generated, but it is not generating correctly.

I restored to a known working backup where I benchmarked the speeds when cpcss is generated for the homepage and active vs not, and sometimes(50/50 chance) generating the cpcss now gets a success message, but the speeds on the frontend do not improve. 

So the issue is twofold, sometimes the specific cpcss generation fails entirely and does not present a success message that it has been generated correctly, and when a success messages **does** appear, the scores on the frontend do not improve at all.
